Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
So, the VFPU in the PSP is a vector floating point processor unit.  Its registers are floats, which happen to also be 32 bits.

Some instructions (not many) treat these registers as integers (e.g. u32 or s32 or even packed other things), and in some cases we interact with them as u32s to simulate floating point behavior.  But their "natural" type is single-precision float.

-[Unknown]